### DISCUSSION TITLE
NOT Fill condition

### DISCUSSION THREAD

**Discussion by darenager** [SynthstromAudible/DelugeFirmware#831] [Ideas]
The addition of the fill function is really superb, an inverse fill option would also be very nice, that is you set a row or event to -FILL and it only plays when FILL isn’t active, this is a great asset when used in combination with fills, for example to drop out the normally playing drums when a fill happens etc.

**Comment by stefanherzog** [comment]
Yes, that would be great. I'm a big fan of this on the Analog Rytm.

In terms of UI, the most straightforward place to add this would be after the `FILL` option, but that would preclude users to add `NO FILL` to notes that also either have a probability or a X-in-Y event. Ideally you'd want to keep that independent. Actually, same goes for `FILL` : would be nice if the fills could also be generative etc. However, I'm not sure where in the UI to add fills or no fills independent of probability or X-in-Y. Any ideas?

**Comment by MasterStatus** [comment]
I came here specifically to suggest this feature 'no fill', so I'm up voting this please!!
